### DISCUSSION TITLE
is it possible to make attendee questions optional

### DISCUSSION THREAD

**Discussion by FiftyBugsOfGrey** [HiEventsDev/Hi.Events#255] [Q&A] [answered]
Is it possible to remove attendee questions..?
So, let's say a group of friends is buying ticket for himself and ten other friends.
Right now, he must enter the name and email of each of the friends, making the check out process very tedious.

Is there any way to make it optional, that is, only the person booking the tickets has to enter his details, but entering the details of his friends who are also attending the event is not mandatory. If it's possible, I am unable to find that setting.

I know there is an option to copy the same details for all attendees but I want to make it optional.

**Comment by daveearley** [comment, marked as answer]
Hi [@FiftyBugsOfGrey](https://github.com/FiftyBugsOfGrey),

No, it is not currently possible to remove the questions. It likely will be an option in the future, but there's no timeline on when.

As a workaround people can use the "Copy details to all attendees" feature, which means you only have to enter the details once.

![Screenshot_20241015-152204](https://github.com/user-attachments/assets/166d71e5-8bbf-49e2-b40c-32c03bed9054)
